src/components/Clock/index.tsx

```diff
@@ -0,0 +1,31 @@
+import React from "react";
+import { ClockContainer, ButtonContainer, ResetContainer } from "./styles";
+import Counter from "../Counter";
+import Button from "../Button";
+
+class Clock extends React.Component {
+  startCounting = () => {
+    console.log("teste");
+  };
+
+  render() {
+    return (
+      <ClockContainer>
+        <ResetContainer>
```

Comment:
[suggestion] You don't need this wrapper around the reset button, so you can just get rid of it.